### DISCUSSION TITLE
D1B1FF37-DDB3-4505-A408-52FCD625632A

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#185] [General]
# D1B1FF37-DDB3-4505-A408-52FCD625632A

Gliese 96, also known as GJ 96, is a red dwarf star located 38.91578 light-years from Earth, and in 2018, a planet the size of Neptune was discovered in an orbit of 73.94 days, and its mass is 19.66 times that of Earth

https://celestia.mobi/addon?item=D1B1FF37-DDB3-4505-A408-52FCD625632A

**Comment by SevenSpheres** [comment]
This planet is challenged by [Ribas et al. 2023](https://ui.adsabs.harvard.edu/abs/2023A%26A...670A.139R/abstract); it is not detected in CARMENES data.
